precommit/src/main/shell/test-patch.sh

```diff
@@ -2328,6 +2328,11 @@ function bugsystem_linecomments
   done < "${fn}"
 }
 
+function trigger_errors
+{
+  echo ${unprotected}
```

Comment:
shellcheck: 8: note: Double quote to prevent globbing and word splitting. [SC2086]